(venv) ubuntu@hotel-management-server:~/hotel-management-backend$ python manage.py makemigrations staff
python manage.py makemigrations tables
python manage.py makemigrations bills
python manage.py makemigrations kitchen
python manage.py migrate
Traceback (most recent call last):
  File "/home/ubuntu/hotel-management-backend/manage.py", line 20, in <module>
    main()
  File "/home/ubuntu/hotel-management-backend/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/hotel-management-backend/apps/kitchen/models.py", line 4, in <module>
    from apps.tables.models import Table
ImportError: cannot import name 'Table' from 'apps.tables.models' (/home/ubuntu/hotel-management-backend/apps/tables/models.py)
Traceback (most recent call last):
  File "/home/ubuntu/hotel-management-backend/manage.py", line 20, in <module>
    main()
  File "/home/ubuntu/hotel-management-backend/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/hotel-management-backend/apps/kitchen/models.py", line 4, in <module>
    from apps.tables.models import Table
ImportError: cannot import name 'Table' from 'apps.tables.models' (/home/ubuntu/hotel-management-backend/apps/tables/models.py)
Traceback (most recent call last):
  File "/home/ubuntu/hotel-management-backend/manage.py", line 20, in <module>
    main()
  File "/home/ubuntu/hotel-management-backend/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/hotel-management-backend/apps/kitchen/models.py", line 4, in <module>
    from apps.tables.models import Table
ImportError: cannot import name 'Table' from 'apps.tables.models' (/home/ubuntu/hotel-management-backend/apps/tables/models.py)
Traceback (most recent call last):
  File "/home/ubuntu/hotel-management-backend/manage.py", line 20, in <module>
    main()
  File "/home/ubuntu/hotel-management-backend/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/hotel-management-backend/apps/kitchen/models.py", line 4, in <module>
    from apps.tables.models import Table
ImportError: cannot import name 'Table' from 'apps.tables.models' (/home/ubuntu/hotel-management-backend/apps/tables/models.py)
Traceback (most recent call last):
  File "/home/ubuntu/hotel-management-backend/manage.py", line 20, in <module>
    main()
  File "/home/ubuntu/hotel-management-backend/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/ubuntu/hotel-management-backend/venv/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/hotel-management-backend/apps/kitchen/models.py", line 4, in <module>
    from apps.tables.models import Table
ImportError: cannot import name 'Table' from 'apps.tables.models' (/home/ubuntu/hotel-management-backend/apps/tables/models.py)
(venv) ubuntu@hotel-management-server:~/hotel-management-backend$ cd
(venv) ubuntu@hotel-management-server:~$ cd hotel-management-backend
(venv) ubuntu@hotel-management-server:~/hotel-management-backend$ deactivate
ubuntu@hotel-management-server:~/hotel-management-backend$ cd
